dunwoody_disaster/views/creditScreen.py

```diff
@@ -1,75 +1,134 @@
+        self.update()  # Update the widget to repaint
 
     def paintEvent(self, event):
+        # Check if there are no more lines to display and return early if true
+        if self.current_line >= len(self.text_lines):
+            return
+
+        # Unpack the current line to get the name and description
+        name, description = self.text_lines[self.current_line]
+
+        # Initialize QPainter for drawing operations
         painter = QPainter(self)
+        # Set rendering hints for smoother drawing
         painter.setRenderHint(QPainter.Antialiasing)
+        # Set the brush color to black for filling the background
         painter.setBrush(Qt.black)
+        # Fill the entire widget with a black background
         painter.fillRect(self.rect(), Qt.black)
+        # Set the opacity for the fade effect
+        painter.setOpacity(self.opacity)
 
-        font = QFont("Arial", 20)
-        font.setBold(True)
+        # Load and scale the image associated with the current name to half the screen width
+        image = QPixmap(self.images[name])
+        image = image.scaled(
+            self.width() // 2,
+            self.height(),
+            Qt.KeepAspectRatio,
+            Qt.SmoothTransformation,
+        )
+        # Draw the image aligned to the left side, vertically centered
+        painter.drawPixmap(0, (self.height() - image.height()) / 2, image)
+
+        # Set the font for drawing text
+        font = QFont("Arial", 24)
         painter.setFont(font)
+        # Set the pen color to white for text drawing
         painter.setPen(QColor(255, 255, 255))
 
-        y = self.height() - self.scroll_position
+        # Calculate the start and width of the area where text will be centered (right half of the screen)
+        text_area_start = self.width() // 2
+        text_area_width = self.width() // 2
+
+        # Calculate the width of the text to center it within the right half
+        name_width = painter.fontMetrics().horizontalAdvance(name)
+        description_width = painter.fontMetrics().horizontalAdvance(description)
+        # Calculate the x-coordinate for the name and description to center them
+        name_x = text_area_start + (text_area_width - name_width) // 2
+        description_x = text_area_start + (text_area_width - description_width) // 2
+        # Determine the vertical positions for the name and description
+        name_y = (self.height() // 2) - 30
+        desc_y = (self.height() // 2) + 30
+
+        # Draw the name and description centered on the right half of the screen
+        painter.drawText(name_x, name_y, name)
+        painter.drawText(description_x, desc_y, description)
 
-        for line in self.text_lines:
-            text_width = painter.fontMetrics().horizontalAdvance(line)
-            x = (self.width() - text_width) / 2
-            painter.drawText(x, y, line)
-            y += self.line_spacing
+    def initSound(self):
+        pygame.mixer.init()
```

Comment:
Wasn't this already inited in `main.py`? I'm not sure how the pygame mixer works, is this going to cause problems? Do we need to make an AudioHandler class to control what audio is playing?

Reply:
As of now I'm currently using the pygame mixer as a multiswitch tool. 

e.g; load this audio -> play this audio -> upon switch turn off this audio -> load that audio -> play that audio -> repeat where necessary.